schedule/microos/stack_tests_python.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: stack_tests_python
+schedule:
+  - microos/disk_boot
+  - '{{add_phub_extension}}'
+  - console/python3_pynacl
+conditional_schedule:
+  add_phub_extension:
+    DISTRI:
+      "sle-micro":
+        - console/add_phub_extension
```

Comment:
So, Microos and SLE Micro have a different scheduling mechanism. Lets leave both of them out in this round, and tackle the test coverage towards Nov/Dec

Reply:
https://openqa.opensuse.org/tests/4575790#

Okay, will remove for now :)

Reply:
@volodymyrkatkalov lets open a bug for python313, and we can think of a way to test the last X versions of Python.

In the meantime: @DimStar77 how would you like to approach cases like this? where we're looking at multiple python versions overlapping, and missing packages, I see https://devguide.python.org/versions/ that 3.9 is still supported, we'd have the same support matrix for SLE I guess:

![grafik](https://github.com/user-attachments/assets/244e7c75-23f0-4b89-9764-1ae7476a5897)

Question is: would we like to test all supported versions by Leap/SLE? at least the basics (virtualenv, pipx) but what about the rest or L3 supported packages? cc @mcepl

Reply:
In the context of Factory we distinguish python versions between 'interpreter-only' provided and 'full stack provided'

i.e Factory currently has provider-only for 3.9 and 3.13 (soon 3.14), and full stack for 3.10, 3.11 (distro default version) and 3.12

Reply:
> In the context of Factory we distinguish python versions between 'interpreter-only' provided and 'full stack provided'
> 
> i.e Factory currently has provider-only for 3.9 and 3.13 (soon 3.14), and full stack for 3.10, 3.11 (distro default version) and 3.12

Can you show me where can I see that? I guess it's a project's setting? - maybe we can find something that requires less busy work of updating test settings.

Reply:
```
> osc meta prjconf openSUSE:Factory | grep pythons
# order of %pythons is important: The last flavor overrides any operation on conflicting files and definitions during expansions,
%pythons %{?!skip_python3:%{?!skip_python310:python310} %{?!skip_python312:python312} %{?!skip_python311:python311}}
```

or `rpm -E %pythons` should give you the versions that have a full stack - this requires python-rpm-macros to be installed though